go.mod

```
@@ -1,11 +1,14 @@
 module github.com/git-town/git-town/v16
 
-go 1.22
+go 1.22.0
```

Apologies for bothering you about this again. Your change on this line sets the Go version on CI to 1.22.0, which triggers the govulncheck warning again. 

```suggestion
go 1.22.9
```

---

done